packages/swift/Sources/Generated.swift

```diff
@@ -235,7 +235,26 @@ class WidgetEventDispatcher: Dispatcher {
         self.delegate = delegate
     }
 
-    func dispatch(_ url: URL) {}
+    func dispatch(_ url: URL) {
+        guard let event = parse(url) else {
+            return
+        }
+
+        delegate.widgetEvent(event)
```

Comment:
`event` is of type `Event`, so Swift correctly calls the `widgetEvent(_: Event)` method. This is the generic "we got a message event".

Reply:
Looking at this and I'm realizing that this generic "we got a message" event is susceptible to errors. In the TS implementation we dispatch this callback before we parse the message, so in the case that fails we still tell the host of the event. I'm gonna either add another `widgetEvent` overload that we can pass the `URL` to (before parsing it) or replace this with that implementation rather than adding another method. Not sure yet, but that'll come in a future PR.